### DISCUSSION TITLE
请问puerts可以结合网页一起使用吗

### DISCUSSION THREAD

**Discussion by SteveSantoso** [Tencent/puerts#1467] [General]
之前网页和UE的交互都是通过通讯来完成的，现在想通过puerts完成，但测试错误了，在网页上运行直接报错，找不到对应的包

**Comment by chexiongsheng** [comment]
你的浏览器又有没使用puerts。
你把浏览器的源码修改下，把js部分替换成用Puerts的FJsEnv就可以。

**Reply by SteveSantoso** [reply]
好的，大致了解l

**Reply by chexiongsheng** [reply]
我们内部一个嵌入到游戏引擎的h5实现（可以理解为嵌入式浏览器），就是通过集成puerts实现js里既能调用浏览器api，也能调用游戏引擎api。
